hack/lib/start.sh

```diff
@@ -522,15 +522,11 @@ function os::start::internal::openshift_executable() {
 
 		openshift_executable="${sudo} docker run ${docker_options} ${volumes} ${envvars} openshift/origin:${version}"
 	else
-		local envvars=""
-		if [[ -n "${ENV:-}" ]]; then
-			envvars="env "
-			for envvar in "${ENV[@]}"; do
-				envvars+="${envvar} "
-			done
-		fi
-
-		openshift_executable="${sudo} ${envvars} $(which openshift)"
+		if [[ -n "${sudo}" ]]; then
+		    openshift_executable="${sudo} -E $(which openshift)"
```

Comment:
@stevekuznetsov how about this instead? https://www.sudo.ws/man/1.8.3/sudo.man.html 

```
-E
The -E (preserve environment) option indicates to the security policy that the user wishes to preserve their existing environment variables. The security policy may return an error if the -E option is specified and the user does not have permission to preserve the environment.


```